src/goto-cc/gcc_cmdline.cpp

```diff
@@ -256,16 +257,18 @@ bool gcc_cmdlinet::parse_arguments(
     if(has_prefix(argv_i, "@"))
     {
       std::ifstream opts_file(argv_i.substr(1));
+      std::ostringstream all_lines;
       std::string line;
 
       while(std::getline(opts_file, line))
-      {
-        // erase leading whitespace
-        line.erase(0, line.find_first_not_of("\t "));
+        all_lines << ' ' << line;
```

Comment:
So this will effectively be replacing newlines with single spaces... it may be an edge case that we really don't care about too much, but just to check have you considered how this may/may not play with the quoting and escaping rules? for @files ? It looks to me like theoretically a "backslash, newline" sequence is valid and should be preserved as a newline, for instance.

Reply:
That's a good call-out, but I'm not entirely sure how where I might be able to put such a newline. The following won't work with GCC:
```
-o
file.gb
-Dhello="  \
"
test.c
```
With the above, `hello` just expands to an empty token. I also tried
```
-o
file.gb
'-Dhello="  \
"'
test.c
```
but then I even get a warning:
```
<command-line>:0:7: warning: missing terminating " character
```
Any ideas what else I might try?

Reply:
Thats good enough for me to think that "theoretically" doesn't map to "actually supported by gcc in reality" :-)